stellarphot/differential_photometry/aij_rel_fluxes.py

```diff
@@ -47,7 +48,7 @@ def calc_aij_relative_flux(star_data, comp_stars,
 
     counts_column_name : str,  optional
         If provided, use this column to find counts.
-        If not provided, the column 'aperture_net_counts' is used.
+        If not provided, the column 'aperture_sum' is used.
```

Comment:
This appears to be inconsistent with line 20 above which has the default as actually 'aperture_net_cnts'.